analytical_engine/java/grape-rdd-reader/pom.xml

```diff
@@ -83,7 +83,6 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>31.0.1-android</version>
         </dependency>
```

Comment:
Are you missing a line here? 
`<version>${guava.version}</version>`

Reply:
The version control of all dependencies in child module can be uplift to parent module, so we don't need to specify the version here, let me add one more commit.